tapiro/core/src/main/scala/io/buildo/tapiro/Meta.scala

```diff
@@ -60,10 +60,13 @@ object Meta {
       case MetarpheusType.Name(name)     => name
     }
 
-  val toScalametaType: MetarpheusType => Type = {
-    case MetarpheusType.Apply(name, args) =>
-      Type.Apply(Type.Name(name), args.map(toScalametaType).toList)
-    case MetarpheusType.Name(name) => Type.Name(name)
+  def toScalametaType(metarpheusType: MetarpheusType, required: Boolean = true): Type = metarpheusType match {
```

Comment:
This is fine, but I think maybe we could instead keep `toScalametaType` as it was before and add a separate function to handle the `required` field, something like:
```scala
def paramTypeToScalametaType(param: RouteParam): Type = {
  val t = toScalametaType(param.tpe)
  if (param.required) t else t"Option[$t]"
}
```

I'm saying this because I think `toScalametaType` makes sense to translate types in general, while handling `required` is only about route params, not other types. And the logic to add `Option` would not be duplicated for both cases of `MetarpheusType`. (It should also means you don't need to expand `.map(toScalametaType)`.)

What do you think?

Reply:
`t"Option[$t]"` is dope!

Anyway,
* d6e540a257246a2c48bb2f93e52b41ac5d883d61: I left `toScalametaType` alone. I created a new function `routeParamToScalametaType` instead.
* 2b2c6690ec970e4493e32ffeaa3d5d49bd78322d: I renamed `toScalametaType` to `metarpheusTypeToScalametaType`

Feel free to rebase and drop the last commit before merge. I though it was nice but course it's not necessary.